### DISCUSSION TITLE
is it possible to search multiple words?

### DISCUSSION THREAD

**Discussion by AnkurTank** [qcz/vscode-text-power-tools#64] [General] [answered]
like grep would support searching multiple words as if "foo\|bar" , is it possible to do the same with text power tools?
is it possible to use other options of grep like case-insensitivity and word match etc?

**Comment by qcz** [comment, marked as answer]
@AnkurTank 

> like grep would support searching multiple words as if "foo|bar" , is it possible to do the same with text power tools?

Yeah, you can use the `Filter lines matching a regex (grep)` command for that.

> is it possible to use other options of grep like case-insensitivity and word match etc?

The command mentioned above is case insensitive. You can do word match with the regex `\b` modifier, for example `\b(foo|bar)\b` or `\bfoo\b|\bbar\b`.

**Reply by AnkurTank** [reply]
Thank you! I was able to get to this after couple of tries.
